Week2/src/databaseBackup.js

```diff
@@ -0,0 +1,9 @@
+import mongoose from "mongoose";
+
+// const server = "mongodb2:27018";
+const server = "127.0.0.1:27018";
+const databaseBackup = "cabify_bootcamp_backup";
+
+export default mongoose.createConnection(`mongodb://${server}/${databaseBackup}`, {
+  useNewUrlParser: true,
```

Comment:
Try to combine `database` and `databaseBackup` into a single function where you pass the databaseName to it